kitchen-tests/cookbooks/base/recipes/packages.rb

```diff
@@ -7,13 +7,7 @@
 
 # this is just a list of package that exist on every O/S we test, and often aren't installed by default.  you don't
 # have to get too clever here, you can delete packages if they don't exist everywhere we test.
-pkgs = %w{lsof tcpdump strace zsh dmidecode ltrace bc curl wget telnet subversion git traceroute htop tmux }
-
-# this deliberately calls the multipackage API N times in order to do one package installation in order to exercise the
-# multipackage cookbook.
-pkgs.each do |pkg|
-  multipackage pkgs
-end
```

Comment:
you need to back this out -- its deliberately being mildly stupid just to exercise the multipackage cookbook to ensure that does not break.  you optimized away the use of that cookbook.